Source/Common/Include/Basics.h

```diff
@@ -230,30 +230,30 @@ struct _strprintf : public std::basic_string<_T>
         return _vscwprintf(format, args);
 #elif defined(__UNIX__)
         // TODO: Really??? Write to file in order to know the length of a string?
-        FILE* dummyf = fopen("/dev/null", "w");
-        if (dummyf == NULL)
-            perror("The following error occurred in basetypes.h:cprintf");
-        int n = vfwprintf(dummyf, format, args);
-        if (n < 0)
-            perror("The following error occurred in basetypes.h:cprintf");
-        fclose(dummyf);
-        return n;
+		const int BUF_SIZE = 256;
+		int n = 0;
```

Comment:
Hard tabs should be replaced with spaces. Currently the merge is blocked in CI because of this.